app/auth/auth-styles.ts

```diff
@@ -0,0 +1,129 @@
+import styled from 'styled-components';
+import COLORS from '../../styles/colors';
```

Comment:
update all imports using the alias! for example: `import COLORS from '@/styles/colors';`